src/main/java/at/hannibal2/skyhanni/features/fishing/FishingAPI.kt

```diff
@@ -87,8 +87,7 @@ object FishingAPI {
     private fun NEUInternalName.isFishingRod() = contains("ROD")
 
     fun ItemStack.isBait(): Boolean {
-        val name = name ?: return false
-        return stackSize == 1 && (name.removeColor().startsWith("Obfuscated") || name.endsWith(" Bait"))
+        return stackSize == 1 && name.removeColor().let { it.startsWith("Obfuscated") || it.endsWith(" Bait") }
```

Comment:
Wouldn't it better to add BAIT to ItemCategory?
Is the Stacksize checking really necessary, if not the implementation would be easy (just a simple pattern)

Reply:
The check if stack size is 1 is necessary to differentiate between a bait that is getting used and fishing up a 32x bait.
Moving to the item category sounds like a good idea still. We just keep having `isUsedBait() = size == 1 && itemCategory == Bait`